packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2022 Google LLC
+ * Copyright 2024 Google LLC
```

Comment:
License only needs to be updated on new files. This should remain 2022.

Reply:
my mistake, i did intend on only updating the new files. not sure how this one slipped through, i'll update